testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/BaseTestHiveCoercion.java

```diff
@@ -152,6 +152,14 @@ protected void doTestHiveCoercion(HiveTableDefinition tableDefinition)
                 "long_decimal_to_varchar",
                 "short_decimal_to_bounded_varchar",
                 "long_decimal_to_bounded_varchar",
+                "varchar_to_tinyint",
+                "string_to_tinyint",
```

Comment:
hmmm... we've unfortunately missed covering these use-cases previously :( 

good catch Praveen.